js/src/views/select2-field.js

```diff
@@ -106,6 +106,7 @@ sui.views.editAttributeSelect2Field = sui.views.editAttributeField.extend( {
 		var $fieldSelect2 = $field[ shortcodeUIData.select2_handle ]({
 			placeholder: "Search",
 			multiple: this.model.get( 'multiple' ),
+			theme: $.fn.select2.defaults.defaults.theme + ' select2-container--shortcake',
```

Comment:
The "theme" parameter gets appended to the string `'select2-container--'` and used as a classname. The default "theme" is "default", so this results in the select2 container getting the classes `'select2-container--default select2-container--shortcake'`. It feels a bit hacky, but it works, and there doesn't seem to be a more straightforward way of targetting only the instance we want, while still allowing it to be styled by the global select2 default css...